tests/app/test_error_handling.py

```diff
@@ -17,6 +19,7 @@ def my_route_handler(param: int, data: Person) -> None:
         assert len(extra) == 3
 
 
+@pytest.mark.xfail()
```

Comment:
In this test, we have route `/{param:int}` and request `/abc`, then test for 400. I think that this should be 404.